src/constants.js

```diff
@@ -1,7 +1,7 @@
-import * as DateUtils from './utils/dateUtils';
-
-const today = DateUtils.getTodaysDateInIso();
-const yesterday = DateUtils.getYesterdaysDateInIso();
+const today = new Date().toISOString();
+let yesterday = new Date();
+yesterday.setDate(yesterday.getDate() - 1);
```

Comment:
Does `yesterday.getDate()` - 1 do what you expect on the 1st of the month?
Better to subtract 24 hours, I would think.

Reply:
yes